metaflow/plugins/secrets/secrets_decorator.py

```diff
@@ -210,8 +210,17 @@ def task_pre_step(
         ubf_context,
         inputs,
     ):
-        if ubf_context == UBF_CONTROL:
-            """control tasks (as used in "unbounded for each") don't need secrets"""
+        # We will skip the secret injection for "locally" launched UBF_CONTROL tasks because otherwise there
```

Comment:
can you add a FIXME for the future that `@parallel` implementation needs to change so that @secrets can work locally for control node in the future.